services/charts_updater.go

```diff
@@ -321,24 +312,26 @@ func stakedEtherChartData() (*types.GenericChartData, error) {
 		return nil, fmt.Errorf("chart-data not available pre-genesis")
 	}
 
-	rows := []struct {
-		Epoch         uint64
-		EligibleEther uint64
+	data := []struct {
+		Indicator string    `db:"indicator"`
+		Time      time.Time `db:"time"`
+		Value     float64   `db:"value"`
 	}{}
 
-	err := db.ReaderDb.Select(&rows, "SELECT epoch, eligibleether FROM epochs ORDER BY epoch")
+	epoch := LatestEpoch()
```

Comment:
Suggestion: Create a function which return LatestEpoch - 1
(easier to read)

Reply:
this is copy paste from all other places in this file, wanted to be consistent

if we use `LatestEpoch() - 1` we still have to check if `LatestEpoch() > 0`

Reply:
fixed